CONTRIBUTING.md

```diff
@@ -331,26 +337,18 @@ If the hook declines your commit, the message will remain in
 Be sure to update your version of the script occasionally as it may evolve as
 our commit guidelines evolve.
 
-## Legal considerations
-
-Please read the [Eclipse Foundation policy on accepting contributions via Git](http://wiki.eclipse.org/Development_Resources/Contributing_via_Git).
-
-Your contribution cannot be accepted unless you have a signed [ECA - Eclipse Foundation Contributor Agreement](https://www.eclipse.org/legal/ECA.php) in place. If you have an active signed Eclipse CLA
-([the CLA was updated by the Eclipse Foundation to become the ECA in August 2016](https://mmilinkov.wordpress.com/2016/08/15/contributor-agreement-update/)),
-then that signed CLA is sufficient. You will have to sign the ECA once your CLA expires.
+## Eclipse OMR Legal Considerations
 
-Here is the checklist for contributions to be _acceptable_:
+### Pull Requests
 
-1. [Create an account at Eclipse](https://dev.eclipse.org/site_login/createaccount.php).
-2. Add your GitHub user name in your account settings.
-3. [Log into the project's portal](https://projects.eclipse.org/) and sign the ["Eclipse ECA"](https://projects.eclipse.org/user/sign/cla).
-4. Ensure that you author the commit using the _same_ email address as your Eclipse account.
-5. Include the appropriate copyright notice and license at the top of each file.
+When a pull request is created (or when new commits are pushed to an existing pull
+request), a webservice will automatically run that will verify that each commit
+author's email address matches an email address that signed the ECA.
+**This service is case-sensitive, so the case of the email addresses must match exactly.**
+The success or failure of this webservice will be reported in the GitHub pull request.
```

Comment:
The only thing I would add here is a more explicit comment that pull requests that fail the ECA check will not be merged.

Reply:
Fixed in forced push.